types/hat/hat-tests.ts

```diff
@@ -0,0 +1,10 @@
+import * as hat from 'hat';
```

Comment:
The `hat` used as function, so it should be `import = require`

Reply:
Done.